script/check_claimed_ideas.pl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env perl
+# check all claimed ideas and make sure they have an IA page.
+# if not then make one
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+use DDGC;
+use Time::Local;
+my $d = DDGC->new;
+
+BEGIN {
+    $ENV{DDGC_IA_AUTOUPDATES} = 1;
+}
+
+MAIN: {
+    my $claimed_without_page = $d->rs('Idea')
+        ->search({
+            claimed_by          => {'!=' => undef},
+            instant_answer_id   => undef,
+        });
+
+    my @time = localtime(time);
+    my $date = "$time[4]/$time[3]/".($time[5]+1900);
```

Comment:
Generally we don't build date strings like that for SQL, since there is potential for ambiguity in day / month ordering and string formatting. There is a DateTime instance "parser" which should take care of these quirks for you:

```perl
my $date = $d->db->format_datetime( DateTime->now );
```

This will also pass a time to the RDBMS but it will truncate it.